third_party/terraform/website/docs/r/storage_bucket.html.markdown

```diff
@@ -39,6 +39,43 @@ resource "google_storage_bucket" "image-store" {
 }
 ```
 
+```hcl
+resource "google_storage_bucket" "image-store" {
+  name     = "image-store.com"
+  location = "US"
+  force_destroy = true
+
+  bucket_policy_only = true
+
+  website {
```

Comment:
there are now 2 examples using the `website` block. It doesn't look like they are different enough use cases to justify both examples so I would collapse them into 1 example that is a super set of the two.

Reply:
Done